16_microbiome_community.Rmd

```diff
@@ -254,6 +258,28 @@ plots <- lapply(k,ClustAndPlot)
 (plots[[1]] + plots[[2]]) / (plots[[3]] + plots[[4]])
 ```
 
+Similarly the _`bluster`_ [@R-bluster] package offers clustering diagnostics
+that can be used for judging the clustering quality (see [Assorted clustering diagnostics](http://bioconductor.org/packages/release/bioc/vignettes/bluster/inst/doc/diagnostics.html)).
+In the following, silhouette width as a diagnostic tool is computed and results are visualized
```

Comment:
I think it should "Silhouette" (capitalized)

Reply:
Can you add citation to the original Silhouette index publication if it can be found?

Reply:
The documentation of `approxSilhouette` does not indicate the publication, only an additional explanation of the approximation; at the source code file. However, there is the following [publication](https://www.sciencedirect.com/science/article/pii/0377042787901257?via%3Dihub), shall I refer to it?

Reply:
The exact[ Silhouette calculation at cluster package](https://www.rdocumentation.org/packages/cluster/versions/2.1.2/topics/silhouette) (referred to by bluster) is using as a reference the same publication above.

Reply:
ok